src/Bigcommerce/Api/Client.php

```diff
@@ -258,7 +258,19 @@ public static function setConnection(Connection $connection = null)
      */
     public static function getCollection($path, $resource = 'Resource')
     {
-        $response = self::connection()->get(self::$api_path . $path);
+        $apiVersion = 2;
+        $api_path = self::$api_path;
+
+        if (strpos($path, '/customers/subscribers') !== false) {
```

Comment:
Since we're passing in the `$resource`, we can clean this up and do `if ($resource === 'Subscriber')`